client/me/help/help-contact/index.jsx

```diff
@@ -260,7 +260,14 @@ module.exports = React.createClass( {
 		}
 
 		if ( ! ( olark.isOlarkReady && sitesInitialized ) ) {
-			return <div className="help-contact__placeholder" />;
+			return (
+				<div className="help-contact__placeholder">
+					<HelpContactForm disabled={true}
+						showSubjectField={ false }
+						onSubmit={ function() {} }
```

Comment:
Ah, cool, I'll update with that when I'm back on later. (And to clarify, it was this particular thing that I meant felt hacky, not the general thing you had proposed, which seems pretty neat.)